charts/values.yaml

```diff
@@ -78,19 +78,33 @@ persistence:
 extraVolumes: []
 # specify extra list of volumeMounts
 extraVolumeMounts: []
-# specify extra list of containerPorts
-extraContainerPorts: []
+
+## Container listener port configuration
```

Comment:
We should explain the difference between "additionContainerPorts" vs "additionalPorts" and why you would set one or the other.

Reply:
We can do it in the documentation but make sure these two values in the values file are in the list of important values file parameters in the README.

Reply:
For consistency you should have as many additionalContainerPorts as you have additionalPorts.
We may change additionalPort to additionalServicePort to make it more explicit.